File: Penjumlahan_1-10.pas
program Penjumlahan;
uses Crt;

var
    i, total: integer;

begin
    ClrScr;
    i := 2;
    total := 1;

    writeln('=============================');
    repeat
        total := total + i;
        i := i + 1;
        writeln('Hasil sementara = ', total);
    until i > 10;

    writeln('------------------------------');
    writeln;
    writeln('Total penjumlahan 1 sampai 10 adalah = ', total);
end.

File: Penjumlahan_1-10.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {
    system("cls");
    cout << "===========================" << endl;
    int total = 1;
    int i = 2;

    do {
        total += i;
        i++;
        cout << "Hasil sementara = " << total << endl;
    } while (i <= 10);

    cout << "--------------------------" << endl;
    cout << "\nTotal penjumlahan 1 sampai 10 adalah = " << total << endl;

    return 0;
}